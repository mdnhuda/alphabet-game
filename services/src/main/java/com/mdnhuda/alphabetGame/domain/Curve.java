package com.mdnhuda.alphabetGame.domain;

import java.util.List;

public class Curve {
    private List<Point> points;

    public List<Point> getPoints() {
        return points;
    }

    public void setPoints(List<Point> points) {
        this.points = points;
    }
}
