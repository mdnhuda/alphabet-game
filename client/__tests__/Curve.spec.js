import React from 'react';
import Curve from '../app/LineOrQuadraticCurve';
import {shallow} from 'enzyme';

describe('LineOrQuadraticCurve', () => {
    it('renders correctly', () => {
        const component = shallow(
            <Curve points={[{"x": 10, "y": 10}, {"x": 10, "y": 20}, {"x": 15, "y": 15}]}
                   stroke={'red'}
                   strokeWidth={3}
            />
        );
        expect(component).toMatchSnapshot();
    });
});
