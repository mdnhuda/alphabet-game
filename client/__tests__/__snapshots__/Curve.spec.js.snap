// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineOrQuadraticCurve renders correctly 1`] = `
<Shape
  sceneFunc={[Function]}
  stroke="red"
  strokeWidth={3}
/>
`;
